backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -700,37 +670,24 @@ public static void initializeBackgroundUtilsCount() {
    */
   public static void registeredBackgroundUtilsCount() {
```

Comment:
`registeredBackgroundUtilsCount` should function should be in common

Reply:
moved